tapiriik/services/GarminConnect/garminconnect.py

```diff
@@ -133,6 +134,7 @@ def __init__(self):
         # Ensure the rate lock file exists (...the easy way)
         open(rate_lock_path, "a").close()
         self._rate_lock = open(rate_lock_path, "r+")
+        '''

```

Comment:
Why do we need to comment this block?

Reply:
My bad, i had to comment it to launch locally tapiriik, i forgot to uncomment